pyuvdata/uvcal/initializers.py

```diff
@@ -66,6 +64,12 @@ def new_uvcal(
         Telescope name.
     cal_style : str
         Calibration style. Options are 'sky' or 'redundant'.
+    freq_array : ndarray of float, optional
+        Array of frequencies in Hz. If given, the freq_range parameter is ignored, and
+        the cal type is assumed to be "gain".
+    freq_range : ndarray of float, optional
+        Frequency ranges in Hz. If given, the freq_array parameter is ignored, and the
+        calibration is assumed to be wide band. The array shape should be (Nspws, 2).
```

Comment:
These both say that if they're set the other one is ignored. Which takes precedence?

Reply:
Good point. I've updated to make it clear that `freq_array` takes precedence.